### PR TITLE
refactor datasource to also work w/o migrations, to let hibernate create the schema itself

### PR DESCRIPTION
**Pull request by MarcoLeipnitz**
The title says it all, an alternative to #100 

---

### Implementation

- [x] Does this code change accomplish what it is supposed to do?
- [x] Can this solution be simplified without unreasonable effort
- [x] Is the code modular enough?
- [x] Does similar functionality already exist in the codebase? If yes, why isn’t it reused?
- [x] Single Responsibility Principle: Does any class serve only one purpose? If not, it should probably split up in multiple classes.
- [x] Interface Segregation: If interface implemented, is it fully implemented?
- [x] Dependency Injection: Are central object instantiation processes managed by a single context?
- [x] If any functionality is missing, is it marked with a comment that starts with TODO:?
- [x] Do you see any potential to improve the performance of the code significantly? 

### Logic Errors and Bugs

- [x] Can you think of any use case in which the code does not behave as intended?
- [x] Can you think of any inputs or external events that could break the code?
- [x] Are edge cases covered by tests?

### Error Handling and Logging

- [x] Is error handling done the correct way?
- [x] Should any logging or debugging information be added or removed?
- [x] Are error messages user-friendly?

### Dependencies (internal and external)

- [x] Were updates to documentation, configuration, or readme files made as required by this change?
- [x] Is the API.md up-to-date?
- [x] Are the enumerations between frontend and backend synchronized?

### Security and Data Privacy

- [x] Does the code introduce any security vulnerabilities?
- [x] Is (user) input validated, sanitized, and escaped to prevent security attacks such as cross-site scripting or SQL injection?
- [x] Is sensitive data like user data handled and stored securely?
- [x] Is the right encryption used?
- [x] Is data retrieved from external APIs or libraries checked for security issues?
- [x] Were any secrets committed?

### Testing and Testability (later)

- [x] Have automated tests been added, or have related ones been updated to cover the change?
- [x] Do the existing tests reasonably cover the code change?
- [x] Are there some test cases, input or edge cases that should be tested in addition?

### Usability and Accessibility

- [x] Is the proposed solution (UI) accessible (e.g. colorblindness, ...)?
- [x] Is the API/UI intuitive to use?
- [x] Was the app tested on another device?

### Readability

- [x] Is the code easy to understand?
- [x] Which parts were confusing to you and why?
- [x] Can the readability of the code be improved by smaller methods/different names?
- [x] Is the code located in the right file/folder?
- [x] Do you think certain methods should be restructured to have a more intuitive control flow?
- [x] Are there redundant or outdated comments?
- [x] Could some comments convey the message better?
- [x] Could some comments be removed by making the code itself more readable?

### CODE REVIEW

File: VLA-Backend/src/main/java/de/vlaorgatu/vlabackend/VlaBackendApplication.java [hunk 1]
```diff
@@ -31,18 +19,5 @@ public static void main(final String[] args) {
...
     }
-
-    /**
-     * Exposes entity IDs in REST endpoints.
-     */
-    // TODO: move this method into a separate config class
-    @Bean
-    public RepositoryRestConfigurer repositoryRestConfigurer() {
-        // source: https://stackoverflow.com/a/70604872
-        return RepositoryRestConfigurer.withConfig(config -> config.exposeIdsFor(
```
Comment (JohaJung) [inline VLA-Backend/src/main/java/de/vlaorgatu/vlabackend/VlaBackendApplication.java]: Don't just remove this, pls. But move it to the DatabaseConfiguration file instead.

File: VLA-Backend/src/main/java/de/vlaorgatu/vlabackend/linusconnection/LinusExperimentBooking.java [hunk 2]
```diff
@@ -15,38 +13,38 @@
...
     @Column(name = "user_id")
-    private Long linusUserId;
+    private int linusUserId;
 
     /**
      * Status of the booking.
      */
     @Column(name = "status")
-    private Long status;
+    private Integer status;
```
Comment (CarolinRoell) [inline VLA-Backend/src/main/java/de/vlaorgatu/vlabackend/linusconnection/LinusExperimentBooking.java]: This is non-null according to #40
